src/components/Authentication/Login.vue

```diff
@@ -115,4 +115,20 @@ export default {
   font-size: 14px;
   padding-top: 1px;
 }
+
+.button-test {
+  color: #ffffff;
+}
+
+#loginButton {
```

Comment:
I see that you repeated this css code in several places. You're still defining the button colors hardcoded, which is not good for maintainability.

Think about this scenario:
- You have repeated this hardcoded color through the whole code.
- Now you want to change the color of all buttons.
- You'll have to update EVERY css file to update the colors.
- This isn't a maintainable approach.


Let's think about this other scenario:
- You've defined a primary color on the Vuetify config file (you can find instructions [here](https://vuetifyjs.com/en/styles/colors#javascript-color-pack))
-  You've used the class `primary` to set all the buttons to the primary color defined previously. Ex:
```vue
<v-btn color="primary">
...
</v-btn>
```
- Now you want to change the color of all buttons.
- The only thing you need to do is update the color in the Vuetify config file. Ex:

```js
export default new Vuetify({
  theme: {
    themes: {
      light: {
        primary: colors.red, // A new color
      },
    },
  },
```